ui/src/vayla-design-lib/layers.scss

```diff
@@ -1,3 +1,3 @@
-$layer-menu: 90;
-$layer-popup: 100;
+$layer-popup: 90;
```

Comment:
editointidialogin tila-dropdown piirtyi joissain tilanteissa dialogin alle

Reply:
Eikös tässä nyt käy niin, että karttatasovalikko tulee dialogin päälle?

Reply:
Tämä sama bugi oli jo korjattu mainiin toisella tiketillä vaihtamalla dropdownin käyttämä z-indeksi `$layer-popup`:iksi. Tämä ja se muutos eivät tule todennäköisesti mergeconflictaamaan, mutta jos jompaakumpaa ei peruuta, niin ollaan käytännössä takaisin alkutilassa ja bugaavissa dropdowneissa.

Reply:
Njoo. Tämä olikin vähän arveluttava fix. Tuollaista asiaa ei voi z-indekseillä ratkaista koska ei popupeilla ja dialogeilla ole mitään kiinteää järjestystä. Kokeilenpa toimiiko tuo nätisti jos ne on tosiaan samassa arvossa.

Reply:
Samalla arvolla tuntuisi toimivan caset mitä keksin. Jos tuossa tulee jotain ongelmaa vielä niin se pitää korjata jotenkin muuten kuin näitä mälväämällä.